### PR TITLE
Fixed typo in string

### PR DESCRIPTION
**Pull request by dsantosp12**

### CODE REVIEW

**Comment by dsantosp12** [comment]
This is just a possible solution for #23. Open to any opinions.

File: Sources/SQLite/SQLite.swift [hunk 1]
```diff
@@ -112,14 +119,6 @@ public class SQLite {
         return Int(id)
     }
 
-    //MARK: Error
-    public enum SQLiteError: Error {
```
Comment (tanner0101) [inline Sources/SQLite/SQLite.swift]: We need to keep this public API around.

**Comment by dsantosp12** [comment]
I think that makes sense! Thank you for the feedback. I will make the changes for 2.3.0 as discussed. 

Do you have any tentative date for 3.0.0 release or this change will trigger it?

**Comment by tanner0101** [comment]
Thanks :) 

No set date for 3.0 yet.

File: Sources/SQLite/SQLite.swift [hunk 2]
```diff
@@ -31,20 +31,21 @@ public class SQLite {
     */
     public init(path: String) throws {
         let options = SQLITE_OPEN_CREATE | SQLITE_OPEN_READWRITE | SQLITE_OPEN_FULLMUTEX
-        if sqlite3_open_v2(path, &database, options, nil) != SQLITE_OK {
-            throw SQLiteError.connection(database?.errorMessage ?? "")
-        }
+
+        let status = sqlite3_open_v2(path, &database, options, nil)
+
+        StatusError.check(with: status, msg: database?.errorMessage ?? "")
```
Comment (0xTim) [inline Sources/SQLite/SQLite.swift]: should this not be `try`? It won't compile for me without it...

Comment (dsantosp12) [reply Sources/SQLite/SQLite.swift]: That is right.

Comment (0xTim) [reply Sources/SQLite/SQLite.swift]: How did pass CI?

Comment (0xTim) [reply Sources/SQLite/SQLite.swift]: (PS don't mean to sound accusatory! Typos are easy but we should just make sure it shouldn't happen again!)

Comment (dsantosp12) [reply Sources/SQLite/SQLite.swift]: That is unknown. In addition, it compiled to me before pushing. Strange. Check #25 with the fix.

Comment (dsantosp12) [reply Sources/SQLite/SQLite.swift]: I agree.

Comment (0xTim) [reply Sources/SQLite/SQLite.swift]: My money's on the good old friend that is derived data! 😁

Comment (dsantosp12) [reply Sources/SQLite/SQLite.swift]: Thanks for the feedback!

**Pull request merged**